OracleDatabase/SingleInstance/extensions/sharding/README.md

```diff
@@ -0,0 +1,7 @@
+# Sharding Extension
+Sharding extension is required to build the catalog and shard containers. When
+the SingleInstance container image is extended with **sharding** extension, it downloads the 
+required scripts from [db-sharding/docker-based-sharding-deployment](https://github.com/oracle/db-sharding/tree/master/docker-based-sharding-deployment)
+repository, and package them with the SingleInstance container image to form extended image.
+
+More information on catalog and shard containers can be found at `db-sharding/docker-based-sharding-deployment` [README](https://github.com/oracle/db-sharding/blob/master/docker-based-sharding-deployment/README.md).
```

Comment:
That README itself is also missing a few "the"'s...

Reply:
@psaini79 Please take a look.

Reply:
Looks good to me